### ISSUE TITLE
Test with Amazon RDS

### ISSUE THREAD

**Issue by ctron** [trustification/trustify-helm-charts#4]

**Comment by gildub** [comment]
@ctron,

On RDS is seems the pg_trgm extension needs to be granted access to the application user.
That would be as part of init-db.sql so it's executed with admin privileges 

```sql
-- grant permission to create extension
GRANT CREATE ON DATABASE :db_name TO :db_user;

-- create the extension
CREATE EXTENSION IF NOT EXISTS pg_trgm WITH SCHEMA public;
```
 
https://github.com/trustification/trustify/blob/0ed6e7b166756bbc08993f1994421247155488a4/migration/src/m0000010_init_up.sql#L12

**Comment by ctron** [comment]
Yes, that makes sense. I guess the right place for this is: https://github.com/trustification/trustify-helm-charts/blob/main/charts/trustify/templates/init/create-database/010-ConfigMap.yaml

However, that also means a change in the `trustify` code base. Might be something to talk to with @chirino.

**Comment by gildub** [comment]
@chirino, let me take back my previous comment. 

I didn't know SeaORM can execute admin level commands. So it can be executed by the migration.

**Comment by gildub** [comment]
@ctron, why not have SeaORM migration take care of database initialization ?

**Comment by gildub** [comment]
@ctron, @chirino, 

I thought the extension had to be created during dbms initialization (init.sql) but granting permission on `trustify` it works :

```sql
GRANT CREATE ON DATABASE :db_name TO :db_user;
```
Interestingly, we have the same issue with vanilla postgres instance, which is if we don't grant such permission then trutd migration will fail.

What was initially tricky is that OCS deployments  is using a container image which seems to not have such restriction (unless there are specific sql statement executed in the init phase we (ansible team) missed).  

Although we now have a workaround for the Ansible playbook (https://github.com/trustification/trustify-ansible).
Meanwhile this grant is way too powerful and need to investigate how to either avoid it or narrow it down to more specific rule.

What do you guys think ?

**Comment by gildub** [comment]
I didn't know of the extra layer of security to better isolate the application user. Which means the migration must be executed with pg admin rights.

So we're now have a consistent behavior no matter what the environment, including RDS.

**Comment by gildub** [comment]
@ctron, We can close this one since RDS works fine.

**Comment by helio-frota** [comment]
> @ctron, We can close this one since RDS works fine.

can we close this ?
